he_twitter_miner/Dockerfile

```diff
@@ -10,4 +10,4 @@ WORKDIR /usr/src/app
 RUN pip install pymongo
 RUN pip install pyyaml
 COPY . .
-CMD [ "python", "./twitterMiner.py" ]
\ No newline at end of file
+CMD [ "python", "./serverStartup.py" ]
```

Comment:
2 things. Python files should be named in `snake_case` by default, and I think `serverStartup.py` is not the best name. Maybe `run_miner.py`. `serverStartup` makes it sound like you're starting a web server or something.